Donation-platform/src/main/webapp/trendingScript.js

```diff
@@ -92,15 +92,16 @@ function updateContOnPage(charities) {
             temp = '';
         }
         cur_count += 1;
-        temp += '<div class="col-3">' + 
-                '<div class="card text-center">' + 
-                '<div class="card-header">Trending #' + cur_count + '</div>' + 
-                '<img class="card-img-top" src=' + charity.imgSrc + ' alt="Card image">' +
-                '<div class="card-body">' +
-                '<h4 class="card-title">' + charity.name + '</h4>' +
-                '<p class="card-text">' + displayTags(charity.categories) + '</p>' + '</div>' +
-                '<div class="card-footer"><a href=' + charity.link + ' target=_blank class="btn btn-primary" role="button">Donate</a></div>' +
-                '</div>' + '</div>';
+        // temp += '<div class="col-3">' + 
```

Comment:
Is this a TODO? Also can we construct HTML using javascript functions instead of innerHTML?

Reply:
I will work on this further when I get to unifying the UI.